src/OVAL/probes/unix/linux/selinuxsecuritycontext_probe.c

```diff
@@ -332,11 +334,11 @@ int selinuxsecuritycontext_probe_main(probe_ctx *ctx, void *arg)
 
 	probe_in  = probe_ctx_getobject(ctx);
 
-	behaviors = probe_obj_getent (probe_in, "behaviors", 1);
-	path      = probe_obj_getent (probe_in, "path",      1);
-	filename  = probe_obj_getent (probe_in, "filename",  1);
-	filepath  = probe_obj_getent (probe_in, "filepath", 1);
-	pid       = probe_obj_getent (probe_in, "pid", 1);
+	behaviors = probe_obj_getent(probe_in, "behaviors", 1);
+	path      = probe_obj_getent(probe_in, "path",      1);
+	filename  = probe_obj_getent(probe_in, "filename",  1);
+	filepath  = probe_obj_getent(probe_in, "filepath",  1);
+	pid       = probe_obj_getent(probe_in, "pid",       1);
```

Comment:
please don't do only formatting changes

Reply:
So, when it is appropriate to fix such formatting error in your opinion?

Reply:
Reverted.